swagger.yml

```diff
@@ -214,3 +220,238 @@ paths:
           description: Not authorized.
         500:
           description: Internal server error.
+
+  /v2/task/<task-id>:
+    get:
+      description: |
+        Read default `stdout` file for the given task. Available on master nodes.
+      parameters:
+        - $ref: "#/parameters/filter"
+        - $ref: "#/parameters/limit"
+        - $ref: "#/parameters/skip"
+        - $ref: "#/parameters/cursor"
+      responses:
+        200:
+          description: Successful response.
+        404:
+          description: Task not found.
+        400:
+          description: Bad request, field not found.
+        401:
+          description: Not authorized.
+        500:
+          description: Internal server error.
+  
+  /v2/task/<task-id>/file/<filename>:
+    get:
+      description: |
+        Read the <filename> for the given task. Available on master nodes.
+      parameters:
+        - $ref: "#/parameters/filter"
+        - $ref: "#/parameters/limit"
+        - $ref: "#/parameters/skip"
+        - $ref: "#/parameters/cursor"
+      responses:
+        200:
+          description: Successful response.
+        404:
+          description: Task not found.
+        400:
+          description: Bad request, field not found.
+        401:
+          description: Not authorized.
+        500:
+          description: Internal server error.
+
+  /v2/component:
+    get:
+      description: |
+        Read the journald logs. Available on all nodes.
```

Comment:
Does this mean all of the journald logs for all components? Maybe it makes sense to clarify here?